README.md

```diff
@@ -1 +1,69 @@
-# Capstone-Project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone-Project
+
+> This project is about creating a website for a company that provides the best tailoring services.
+
```

Comment:
- [ ]  Kyndly add the required loom video 📽️📺 on the pull request description accorging to the project requirements.

![image](https://user-images.githubusercontent.com/95243996/196677892-c2fef130-8a7e-48e7-84b1-7ba2ef6eb0ad.png)

![image](https://user-images.githubusercontent.com/95243996/196677787-ba0bf720-7880-490e-bd5f-0f98d57eadd5.png)